src/components/modals/ReferenceModal.vue

```diff
@@ -113,7 +116,7 @@ h4 {
   text-transform: capitalize;
   display: flex;
   align-items: center;
-  height: 20px;
+  height: 12px;
```

Comment:
That's too crammed. Keep it at 20px please.
![Bildschirmfoto 2021-02-10 um 21 39 58](https://user-images.githubusercontent.com/325521/107569399-8e5d5c80-6be8-11eb-9dc5-1941be9b2573.png)

Reply:
Without some other change this breaks the purpose of maximize for me - because a standard 13/4/4/4 sheet then doesn't fit within 1/2 a screen. I'll see what else might work.

Reply:
Altering padding-top in ul.li.icon from 66% to 65% seems to do the trick, I hope that's still visually OK for you.

Reply:
I'll check it out, but the thing is: you can't optimize the design for a specific screen size if that makes it look bad on other screen sizes. We want to keep the page usable and looking good on as many different resolutions as possible.